src/index.js

```diff
@@ -165,6 +165,12 @@ function generate(dataFrom, config = defaultConfig) {
   let definitions = moduleDefs.map(def => def.code).join('\n').replace(/\n(\s+)\n/g, '\n');
 
   if(postprocessing) definitions = postprocessing(definitions, dataFrom);
+
+	if(collection.filter( issue => issue.type === 'error').length) 
+    throw new Error('Found errors while building definition file:\n' + 
+    JSON.stringify(collection.filter( issue => issue.type === 'error'), null, '  ')
```

Comment:
outputting the "message" will suffice.
Can you clean this up a bit? (tabs/spacing, single use of filter(), etc)

Reply:
You may need to create a custom error to achieve that.

Reply:
I was getting a printout of the error object and then the full stack trace.  It was really hard to find the underlying issue.  I think if only the message were output it would be a lot cleaner.  If the messages have been output then maybe we can throw a custom error which outputs nothing?